_posts/en/newsletters/2023-03-08-newsletter.md

```diff
@@ -0,0 +1,209 @@
+      choosing to spend using the freeze tapscript from the output of
+      either the vaulting or initial unvaulting transactions.  Recall
+      that Alice explicitly placed the freeze tapscript in the vaulting
+      transaction, and note that it was implicitly carried over by the
+      transaction with initiated the unvaulting.
+
+  One of the advantages to users of this approach over the original
+  `OP_VAULT` design is that the freeze tapscript can contain any
+  authorization conditions Alice wants to specify.  In the `OP_VAULT`
+  proposal, anyone knowing the parameters chosen by Alice could spend
+  her funds to the freeze script.  That wasn't a security problem but it
+  could be annoying.  In Sanders's design, Alice could (for example)
+  require a signature from a very lightly protected wallet in order to
+  initiate a freeze---this would perhaps be enough of a burden to
+  prevent most griefing attacks but not enough of a barrier to prevent
+  Alice from quickly freezing her funds in an emergency.
+
+  Several other advantages are aimed at making the consensus-enforced
+  [vaulting protocol][topic vaults] easier to understand and verify as
+  safe.
+
+  As of this writing, Sanders's proposal had only received limited
+  discussion on the mailing list.
+
+- **New Optech Podcast:** the weekly Optech Audio Recap hosted on
+  Twitter Spaces is now available as a podcast.  Each episode will be
+  available on all popular podcast platforms and on the Optech website
+  as a transcript.  For more details, including why we think this is a
+  major step forward in Optech's mission to improve Bitcoin technical
+  communication, please see our [blog post][podcast post].
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[bitcoin-inquisition: Activation logic for testing consensus changes][review club bi-16]
+is a PR by Anthony Towns that improves activation and deactivation
+logic for the [Bitcoin Inquisition][] project, which runs on [signet][topic signet].
+This project was covered in [newsletter #219 bi][].
+
+While most of the [Bitcoin Inquisition PRs][bi prs] implement specific consensus or
+relay policy changes, this PR modifies the Bitcoin Inquisition framework itself.
+
+In contrast to consensus and relay changes on mainnet -- which are difficult
+and time-consuming to activate, requiring the careful building of (human)
+consensus and an elaborate [soft fork activation][topic soft-fork-activation]
+mechanism -- on a test network activating these changes can be streamlined.
+The PR also implements a way to deactivate changes that turn out to be buggy
+or undesired, which is a major departure from mainnet.
+Specifically, this PR replaces [BIP9][] block version bit semantics with what
+are called [Heretical Deployments][].
```

Comment:
I think the paragraph would flow better if this was the first sentence, rather than the last sentence.